statuses.go

```diff
@@ -60,6 +60,18 @@ const (
 	ChannelNotFoundError
 )
 
+func (s Status) IsAccepted() bool {
+	return s != Requested && s != Cancelled && s != Cancelling && s != Failed && s != Failing
+}
+
+func (s Status) IsResponderPaused() bool {
+	return s == ResponderPaused || s == BothPaused || s == Finalizing
+}
+
+func (s Status) InFinalization() bool {
+	return s == Finalizing || s == Completed || s == Completing
```

Comment:
`ResponderFinalizing` doesn't come into play here at all?

Reply:
ResponderFinalizing is the for the initiator, indicating the responder is finalizing. For the moment, we only care about the responder being in a finalization state, as it only affects their logic.

Probably and maybe we should add it to the list but I just want to be careful.

Reply:
I think I'll just go ahead and add it